File: 05. Percabangan/D. If Then Else/d_if_then_else.pas
program d_if_then_else;
uses crt;
var N : longint;
begin
  readln(N);
  if (N > 0) then
    writeln('positif')
  else if (N < 0) then
    writeln('negatif')
  else
    writeln('nol');
end.

File: 05. Percabangan/D. If Then Else/d_if_then_else.java
import java.util.Scanner;

class d_if_then_else {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int N = scan.nextInt();
        scan.close();
        if (N > 0) {
            System.out.println("positif");
        } else if (N < 0) {
            System.out.println("negatif");
        } else {
            System.out.println("nol");
        }
    }
}
